src/JabbR-Core/Commands/CommandManager.cs

```diff
@@ -183,32 +199,32 @@ public void MatchCommand(string commandName, out ICommand command)
             }
         }
 
-        private static IList<ICommand> GetCommands()
+        public static IList<ICommand> GetCommands()
         {
             // Use MEF to locate the content providers in this assembly
             //var catalog = new AssemblyCatalog(typeof(CommandManager).Assembly);
             //var compositionContainer = new CompositionContainer(catalog);
             //return compositionContainer.GetExportedValues<ICommand>().ToList();
-            return new List<ICommand>() { new JoinCommand() };
+            return new List<ICommand>() { new JoinCommand(), new OpenCommand(), new CreateCommand(), new LeaveCommand() };
```

Comment:
@heatherbshapiro is their an update on this? I had work that was manually adding commands as well (like new HelpCommand()) and he said I should maybe not merge it because you've been working on the fix to this?